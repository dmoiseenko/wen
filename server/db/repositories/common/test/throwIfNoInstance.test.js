const throwIfNopInstance = require('../throwIfNoInstance');

it('should throw if instance is null', () => {
  const error = new Error('instance is null');

  function fut() {
    throwIfNopInstance(error)(null)('octopus');
  }

  expect(fut).toThrowErrorMatchingSnapshot();
});
