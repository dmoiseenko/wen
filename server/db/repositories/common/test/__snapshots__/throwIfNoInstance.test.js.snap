// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should throw if instance is null 1`] = `"instance is null"`;
